build/utils/scripts/dappnode_access_credentials.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# PRIORITY: Wi-Wi > Avahi > VPN (Wireguard > OpenVpn)
+
+#############
+#0.VARIABLES#
+#############
+# Containers
+WIFI_CONTAINER="DAppNodeCore-wifi.dnp.dappnode.eth"
+WIREGUARD_CONTAINER="DAppNodeCore-wireguard.dnp.dappnode.eth"
+OPENVPN_CONTAINER="DAppNodeCore-vpn.dnp.dappnode.eth"
+HTTPS_CONTAINER="DAppNodeCore-https.dnp.dappnode.eth"
```

Comment:
Is this the only place where we define container names? If in multiple places consider grouping somewhere if it makes sense